spec/query_spec.rb

```diff
@@ -623,7 +666,6 @@ def test_method
         token = client.query { login @user[:ref], password: @password }
         user_client = get_client secret: token[:secret]
 
-        #fixme: use identity() instead
```

Comment:
How come this was removed and not performed?

Reply:
when I added this comment I thought `identity` would return the token itself, but it's not the case, `identity` returns the instance associated with the token